### PR TITLE
chore: remove 2015 preset and add env

### PR DESCRIPTION
**Pull request by hozefaj**
Is there any requirement in setting a specific set of targets?

### CODE REVIEW

**Comment by timdorr** [comment]
We should still target IE11 support. If not, I would argue that warrants a major bump.

**Comment by hozefaj** [comment]
@timdorr So what's the recommendation, I agree that dropping ie 11 support should warrant a major version bump. 

But on the other hand, supporting ie11 also dings everyone else that use browsers that have over the air updates.

**Comment by timdorr** [comment]
For the moment, I would include IE11. We can revisit the config for the next major release.

File: .babelrc [hunk 1]
```diff
@@ -1,30 +1,6 @@
...
-    "transform-es3-property-literals",
-  ],
-  "env": {
-    "commonjs": {
-      "plugins": [
-        ["transform-es2015-modules-commonjs", { "loose": true }],
-      ]
-    },
+  "presets": ["env"],
+  "targets": {
```
Comment (hozefaj) [inline .babelrc]: https://babeljs.io/docs/en/babel-preset-env#targets

**Comment by hozefaj** [comment]
@timdorr I have added `ie11` as a target. Also removed the lock file. Best that a new lock file is generated before a new version of the package is published.

**Comment by timdorr** [comment]
No, please include the lock file. That shows exactly which deps to install.

**Comment by hozefaj** [comment]
> No, please include the lock file. That shows exactly which deps to install.

Done, added the `package.json-lock` file to the PR

**Comment by timdorr** [comment]
Excellent. Thank you!

**Pull request merged**